examples/src/scala/org/pantsbuild/example/several_scala_targets/BUILD

```diff
@@ -17,3 +17,18 @@ scala_library(
   name='greeting_everybody',
   sources=['greeting_everybody.scala'],
 )
+scala_library(
+  name='greet_json',
+  dependencies=[
+    'examples/src/scala/org/pantsbuild/example/several_scala_targets/node:names_to_greet',
+  ],
+  sources=['greeting_json_names.scala'],
```

Comment:
nit: Scala filenames are usually camel case.

Reply:
I was just following the convention of the names in the several_scala_targets directory 🤔